packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart

```diff
@@ -504,7 +504,7 @@ class GoogleMapsFlutterAndroid extends GoogleMapsFlutterPlatform {
   /// for more information.
   ///
   /// Currently defaults to true, but the default is subject to change.
-  bool useAndroidViewSurface = true;
+  bool useAndroidViewSurface = false;
 
```

Comment:
It doesn't need to be part of this PR, but we should consider doing a follow-up that switches from using `AndroidView` to calling `initSurfaceAndroidView` in the `false` codepath, so that fallback is to HC rather than VD. IIRC there are significant accessibility problems with maps under VD, so HC is probably the better experience as a fallback (and it would be consistent with how the plugin has worked for the last year or so, so it's well-tested).